recipe/meta.yaml

```
@@ -17,22 +17,21 @@ build:
 requirements:
   host:
     - python
-    - setuptools
+    - setuptools >=71
     - pip
-    - wheel
+    - toml
     - setuptools_scm >=8
   run:
     - python
-    - numpy >=1.24.3,<3
-    - scipy >=1.10.1,<2
-    - scikit-learn >=1.3.2,<2
-    - sklearn-compat >=0.1,<1
+    - numpy >=1.25.2,<3
+    - scipy >=1.11.4,<2
+    - scikit-learn >=1.4.2,<2
     - joblib >=1.1.1,<2
```

https://github.com/scikit-learn-contrib/imbalanced-learn/blob/0.14.0/pyproject.toml#L31

```suggestion
    - joblib >=1.2.0,<2
```